apps/src/sharedComponents/footer/CopyrightDialog/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import Link from '@cdo/apps/componentLibrary/link';
+import {BodyThreeText, Heading3} from '@cdo/apps/componentLibrary/typography';
+import {pegasus} from '@cdo/apps/lib/util/urlHelpers';
+import AccessibleDialog from '@cdo/apps/sharedComponents/AccessibleDialog';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import i18n from '@cdo/locale';
+
+import './style.scss';
+import defaultStyle from '@cdo/apps/sharedComponents/accessible-dialogue.module.scss';
+
+export interface CopyrightDialogProps {
+  isOpen: boolean;
+  closeModal: (e: React.ChangeEvent) => void;
+}
+
+const CopyrightDialog: React.FC<CopyrightDialogProps> = ({
+  isOpen,
+  closeModal,
+}) => {
+  return isOpen ? (
+    <AccessibleDialog
+      styles={{xCloseButton: defaultStyle.xCloseButton + ' no-mc'}}
+      id="copyright-modal"
+      className="copyrightDialog"
+      onClose={closeModal}
+      closeOnClickBackdrop={true}
+    >
+      <div id="copyright-modal-content">
+        <div id="copyright-modal-header">
+          <Heading3>{i18n.copyright()}</Heading3>
+        </div>
+        <hr aria-hidden="true" />
+        <div id="copyright-modal-body" className="copyrightScrollArea">
+          <SafeMarkdown
+            markdown={i18n.copyright_thanks({
+              donors_link: pegasus('about/donors'),
+              partners_link: pegasus('about/partners'),
+              team_link: pegasus('about/team'),
+            })}
+          />
+          <BodyThreeText>{i18n.copyright_specialRecognition()}</BodyThreeText>
+          <SafeMarkdown
+            markdown={i18n.copyright_artFrom({
+              current_year: new Date().getFullYear(),
+            })}
+          />
+          <SafeMarkdown
+            markdown={i18n.copyright_codeLicense({
+              gnu_license_link:
+                'https://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html',
+            })}
+          />
+          <BodyThreeText>{i18n.copyright_builtOnGithub()}</BodyThreeText>
+          <Link
+            href="https://aws.amazon.com/what-is-cloud-computing"
+            className="awsLogoContainer"
+          >
+            <img
+              src="/shared/images/Powered-By_logo-horiz_RGB.png"
+              alt="Powered by AWS Cloud Computing"
```

Comment:
Does this need to be translated?

Reply:
Good question!
I don't think it needs to be translated because “Powered by AWS” logo is a trademark of Amazon.com, and we do not translate it in any other instances.